CHANGELOG/CHANGELOG-0.1.md

```diff
@@ -0,0 +1,69 @@
+# Release notes for v0.1.0
+
+## Changes by Kind
+
```

Comment:
This is the first release, so it should include your very first few PRs that introduced this feature.